### DISCUSSION TITLE
Plugins manager (extensions)

### DISCUSSION THREAD

**Discussion by donavan-tangible** [directus/directus#18679] [Feature Requests]
### Summary

Simply add a way to install/remove extensions via the UI. 

### Motivation

Some users may not have experience of developing nodes application.

### Detailed Design

- Add a way in the UI the select a extensions to install
- Install the extension from the UI
- Remove the extensions from the UI

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

That will help admin user.

### Alternatives

A developper in necessary to add an extension. 

### Adoption Strategy

Npm package

**Comment by rijkvanzanten** [comment]
That's what https://github.com/directus/directus/discussions/2820 is all about! 😄
